go.mod

```diff
@@ -144,6 +144,8 @@ require (
 
 replace github.com/altoros/gosigma => github.com/juju/gosigma v0.0.0-20200420012028-063911838a9e
 
+replace github.com/juju/description/v3 => github.com/cderici/description/machine-display-name
```

Comment:
To be changed after juju/description is merged

Reply:
Yes, this is to allow this PR to use another PR on my `description` fork, which is a requirement of this PR and will be landed before, so I'll revert this back as well :+1: